sanic/websocket.py

```diff
@@ -40,11 +768,12 @@ def __init__(
         **kwargs
     ):
         super().__init__(*args, **kwargs)
-        self.websocket = None
+        self.websocket = None # type: Union[None, WebsocketImplProtocol]
         # self.app = None
         self.websocket_timeout = websocket_timeout
         self.websocket_max_size = websocket_max_size
-        self.websocket_max_queue = websocket_max_queue
+        if websocket_max_queue is not None and int(websocket_max_queue) > 0:
+            error_logger.warning(DeprecationWarning("websocket_max_queue is no longer used. No websocket message queueing is implemented."))
```

Comment:
We should standardize on how we want to raise these.

Also, maybe add a code comment to remind use to remove this after 21.12.

Reply:
Yeah, I've looked at other deprecation warnings in Sanic, and there's not really a standardized way of doing it. Perhaps some more discussion is needed.